client/state/data-layer/wpcom/read/lists/index.js

```diff
@@ -48,7 +48,9 @@ registerHandlers( 'state/data-layer/wpcom/read/lists/index.js', {
 					return [
 						receiveReaderList( { list } ),
 						navigate( `/read/list/${ list.owner }/${ list.slug }/edit` ),
-						successNotice( translate( 'List created successfully!' ) ),
+						successNotice( translate( 'List created successfully.' ), {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.